driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -341,7 +408,7 @@ pub mod tests {
     }
 
     #[test]
-    fn serialize_result_fails_if_order_buy_volume_not_parseable() {
+    fn serialize_result_fails_if_order_buy_volume_not_parsable() {
```

Comment:
Seems unrelated to the PR. Hehe, I've noticed that incorrect spelling in function names and comments really gets to you :stuck_out_tongue: